Atomic/help.py

```diff
@@ -0,0 +1,41 @@
+from . import Atomic
+import subprocess
+from pydoc import pager
+from os import environ
+from . import mount
+import sys
+
+class AtomicHelp(Atomic):
+    def help(self):
+        """
+        Displays the image/containe help file in a pager
+        :return: None
+        """
+        # Set the pager to less -R
+        dm = mount.DockerMount("/tmp")
+        enc=sys.getdefaultencoding()
```

Comment:
the DockerMount class requires a mount point for mounting images/containers.  The mount point is not used until something is actually mounted, so the passing of "/tmp" is quite benign.  I'm happy to change to '/run' if you prefer.